### PR TITLE
fix: connect wallet

### PR DESCRIPTION
**Pull request by ezenwankwogabriel**
> Creating a release? Please use the [Release PR Template](https://github.com/WalletConnect/walletconnect-monorepo/blob/v2.0/.github/pr_template_release.md) instead.

## Description

Fixes error on connecting wallet from the package bumps

Please include a brief summary of the change.

## How has this been tested?

Please describe the tests that you ran to verify your changes. Please also list any relevant details for your configuration.

- [x] Link to mobile
- [x] can connect wallet using wallet-connect 

## Fixes/Resolves (Optional)

- error from using pino logger
- do not override topic

Please list any issues that are fixed by this PR in the format `#<issue number>`. If it fixes multiple issues, please put each issue in a separate line. If this Pull Request does not fix any issues, please delete this section.

## Examples/Screenshots (Optional)

Please attach a screenshot or screen recording of features/fixes you've implemented to verify your changes. Please also note any relevant details for your configuration. If your changes do not affect the UI, please delete this section.

Use this table template to show examples of your changes:

| Before       | After        |
| ------------ | ------------ |
| Content Cell | Content Cell |
| Content Cell | Content Cell |

## Checklist

- [ ] I have performed a self-review of my own code
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

# Additional Information (Optional)

Please include any additional information that may be useful for the reviewer.

### CODE REVIEW

File: packages/utils/src/network.ts [hunk 1]
```diff
@@ -26,6 +26,7 @@ export function getBrowserOnlineStatus() {
 export async function getReactNativeOnlineStatus(): Promise<boolean> {
   // global.NetInfo is set in react-native-compat
   if (isReactNative() && typeof global !== "undefined" && (global as any)?.NetInfo) {
+    return true; // TODO: discuss on adding react-native-community/netinfo
```
Comment (ezenwankwogabriel) [inline packages/utils/src/network.ts]: @matias-la [react-native-community/netinfo](https://github.com/ExodusMovement/walletconnect-monorepo/blob/7a14ce2b01bf141b65408119de6dfc9e1eb32f24/packages/react-native-compat/index.js#L37)  is needed for this. what'd be best action here ?

Comment (ezenwankwogabriel) [reply packages/utils/src/network.ts]: repace with `@exodus/netinfo` ?

Comment (633kh4ck) [reply packages/utils/src/network.ts]: @ezenwankwogabriel yeah, we have `@exodus/netinfo@1.4.2` audited - you can give it a try

**Review by matias-la — APPROVED** [review]
utACK

Agree that we should use the `netinfo` fork, or directly to just get rid of the dependency

**Review by 633kh4ck — APPROVED** [review]
utACK

**Pull request merged**